jwst/datamodels/vegamag_offset.py

```diff
@@ -0,0 +1,85 @@
+from .reference import ReferenceFileModel
+
+__all__ = ['FgsImgVegaMagOffsetModel', 'MirImgVegaMagOffsetModel',
+           'NrcImgVegaMagOffsetModel', 'NisImgVegaMagOffsetModel']
+
+
+class FgsImgVegaMagOffsetModel(ReferenceFileModel):
+    """
+    A FGS imaging data model for reference files containing Vega
+    magnitude offsets from AB magnitudes.
+
+    Parameters
+    ----------
+    vegamag_table : numpy table
+        Vega magnitude offsets tables.
+        A table-like object containing row selection criteria made up of
+        instrument mode parameters and aperture correction factors
```

Comment:
Whoops - copy-n-paste error: these tables don't contain aperture correction factors. Need to update (in all schemas).

Reply:
Fixed.